File: apps/s-2-admin/src/api/schedule/ScheduleUpdateInput.ts
import { UserWhereUniqueInput } from "../user/UserWhereUniqueInput";
import { EventTypeUpdateManyWithoutSchedulesInput } from "./EventTypeUpdateManyWithoutSchedulesInput";
import { AvailabilityUpdateManyWithoutSchedulesInput } from "./AvailabilityUpdateManyWithoutSchedulesInput";

export type ScheduleUpdateInput = {
  user?: UserWhereUniqueInput;
  name?: string;
  timeZone?: string | null;
  eventType?: EventTypeUpdateManyWithoutSchedulesInput;
  availability?: AvailabilityUpdateManyWithoutSchedulesInput;
};

File: apps/cactus-demo-dotnet/src/APIs/Schedule/Dtos/ScheduleUpdateInput.cs
namespace CactusDemoDotnet.APIs.Dtos;

public class ScheduleUpdateInput
{
    public int? Id { get; set; }

    public int? User { get; set; }

    public string? Name { get; set; }

    public string? TimeZone { get; set; }

    public List<int>? EventType { get; set; }

    public List<int>? Availability { get; set; }
}
